SoftLayer/CLI/virt/create.py

```diff
@@ -334,6 +335,20 @@ def cli(env, **args):
     env.fout(output)
 
 
+def get_total_recurring_fee(args, result, table, total_hourly, total_monthly):
```

Comment:
I don't like functions that modify the arguments that get passed in, good way to introduce bugs in my opinion. 

Lets change this function to `_build_reciept_table(prices)` which will build and return the whole table used to print costs. It doesn't need to know about hourly v monthly.

```python
def get_total_recurring_fee(prices, billing="hourly"):
    """Retrieve the total recurring fee of the items prices"""
    total = 0.0
    table = formatting.Table(['Cost', 'Item'])
    table.align['Cost'] = 'r'
    table.align['Item'] = 'l'
    for price in prices:
        rate = 0.0
        if billing == "hourly":
            rate += float(price.get('hourlyRecurringFee', 0.0))
        else:
            rate += float(price.get('recurringFee', 0.0))
        total += rate

        table.add_row(["%.3f" % rate, price['item']['description']])
    table.add_row(["%.3f" % total, "Total %s cost" % billing])
    return table
```

**prices need to be to 3 decimals**

Reply:
Done.